### DISCUSSION TITLE
🖋: プロキシ環境下のAndroid Emulatorで外部アクセスする場合のプロキシ設定

### DISCUSSION THREAD

**Discussion by takei0w0** [ws-4020/mobile-app-crib-notes#123] [Ideas]
プロキシ環境下のAndroid Emulatorで外部にアクセスする場合Settingsからプロキシ設定が必要になる。
ただし、React Nativeのbundleサーバーはローカル環境のためプロキシを通さないようにしたい。

Wi-Fiのプロキシ設定で除外対象のホストを指定することで解決できる。

> 1. Settings > Network & internet > Wi-Fi
> 2. AndroidWifiをタップ
> 3. 右上の鉛筆アイコンをタップ
> 4. ProxyでManualを選択
> 5. 以下の通り設定
>    *  Proxy hostname: proxy.example.com
>    *  Proxy port: 8080
>    *  Bypass proxy for: 除外対象のホスト（今回の場合10.0.2.2）
> 6. Saveをタップ

※Android8はWi-Fi名を長押しすると設定メニューが出る
※Android7以下で設定できるか確認できていない

**Comment by creature-water-valley** [comment]
[基本的なセットアップ](https://ws-4020.github.io/mobile-app-crib-notes/react-native/santoku/development/development-environment/basic-setup)に追加済み
